cmd/containerd/command/service_windows.go

```diff
@@ -142,13 +171,39 @@ func registerService() error {
 		DisplayName:  "Containerd",
 		Description:  "Container runtime",
 	}
-
+	containerdProcessHandle := windows.CurrentProcess()
```

Comment:
Can we call this directly in `SetPriorityClass` instead.

Reply:
Yes, we can do that.